modules/c/examples/code_snippets/main.c

```diff
@@ -1078,3 +1197,392 @@ Placeholder for Date accessors.
 // tag::fts-index[]
 // placeholder
```

Comment:
I think I've added this in a later commit, FYI

Reply:
Yes indeed, you did. Thanks